web/views.py

```
@@ -3788,6 +3793,154 @@ def get_object(self):
+                new_slug = base_slug
+                counter = 1
+
+                while Meetup.objects.filter(slug=new_slug).exclude(pk=meetup.pk).exists():
+                    new_slug = f"{base_slug}-{counter}"
+                    counter += 1
+
+                # Update the slug
+                form.instance.slug = new_slug
+            form.save()
+            return redirect("meetup_detail", slug=meetup.slug)
+        else:
```

Ruff (0.8.2)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)